MultiplayerMod/MultiplayerMod.csproj

```diff
@@ -57,4 +57,7 @@
         <Reference Include="UnityEngine.IMGUIModule" HintPath="$(ManagedPath)\UnityEngine.IMGUIModule.dll" Private="false" />
         <Reference Include="ImGui.NET" HintPath="$(ManagedPath)\ImGui.NET.dll" Private="false" />
     </ItemGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\MultiplayerMod.AttributeProcessor\MultiplayerMod.AttributeProcessor.csproj" Private="false"/>
```

Comment:
Move to a separate repo and use as a nuget dependency?

Reply:
Okay, let's see what can be done here.

Reply:
I started to create packages and didn't get the reason for this to be in a separate repo and published to nuget. It's highly-coupled with the mod itself. I'm not sure it's worth it to decouple it and support different target frameworks, etc. Maybe it is, but later I guess. We should discuss it later anyway.